<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KGraphViewerPreferencesOpenInExistingWindowWidget</class>
 <widget class="QWidget" name="KGraphViewerPreferencesOpenInExistingWindowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>197</width>
    <height>128</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="openInExistingWindowMode">
     <property name="title">
      <string>Open in existing window</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="yes">
        <property name="text">
         <string>&amp;Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="no">
        <property name="text">
         <string>&amp;No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ask">
        <property name="text">
         <string>As&amp;k</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
